android/build.gradle

```diff
@@ -10,13 +10,21 @@ buildscript {
 
 apply plugin: 'com.android.library'
 
+def _ext = rootProject.ext
```

Comment:
Why the `_` prefix for all variables, can you rename the variables to something more sensible?